crates/dwallet-mpc-centralized-party/src/lib.rs

```diff
@@ -374,3 +392,8 @@ fn cg_secp256k1_public_key_share_from_secret_share(
         )?;
     Ok(generator_group_element.scale(&Uint::<{ SCALAR_LIMBS }>::from_be_slice(&secret_key_share)))
 }
+
+/// Derives [`DWalletPublicKeys`] from the given dwallet DKG output.
+pub fn public_keys_from_dwallet_output(output: Vec<u8>) -> anyhow::Result<Vec<u8>> {
```

Comment:
```suggestion
pub fn public_keys_from_dkg_output(output: Vec<u8>) -> anyhow::Result<Vec<u8>> {
```
keep the name consistent